nautobot/dcim/api/serializers.py

```diff
@@ -1069,6 +1072,16 @@ class Meta:
 
     def validate(self, data):
 
+        # set interface status to active if status not provided
+        if not data.get("status"):
+            # status is currently required in the Interface model but not required in api_version < 1.3 serializers
+            # which raises an error when validating except status is explicitly set here
+            query = Status.objects.get_for_model(Interface)
+            try:
+                data["status"] = query.get(slug=InterfaceStatusChoices.STATUS_ACTIVE)
+            except Status.DoesNotExist:
+                data["status"] = query.first()
```

Comment:
To @jathanism's comment earlier - in the case where Active status doesn't exist, better to have a fallback to the first (arbitrary) status and log a warning? Or better to report an error and require that the user either recreate the Active status or switch to the newer API version that lets them explicitly choose a status? I know I suggested the fallback originally but now I'm reconsidering - sorry!